apollo-router/src/plugins/telemetry/metrics/prometheus.rs

```diff
@@ -45,7 +68,7 @@ impl MetricsConfigurator for Config {
                 ))
                 .try_init()?;
             builder = builder.with_custom_endpoint(
-                "/prometheus",
+                "",
```

Comment:
The default path is now [ /metrics](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config), which matches the default prometheus behavior.

Good call though, because it prompts me to add a BREAKING section to the changelog